packages/@cdktf/hcl2cdk/test/expressionToTs.test.ts

```diff
@@ -198,37 +270,53 @@ describe("expressionToTs", () => {
   test("detects splat reference within function", async () => {
     const expression = "${toset(aws_s3_bucket.examplebucket.*)}";
     const scope = getScope({ resources: ["aws_s3_bucket.examplebucket"] });
-    const result = await convertTerraformExpressionToTs(expression, scope, "");
+    const result = await convertTerraformExpressionToTs(
+      expression,
+      scope,
+      getType
+    );
     expect(code(result)).toMatchInlineSnapshot(
-      `"cdktf.Fn.toset(\\"\${\\" + awsS3BucketExamplebucket.fqn + \\"}.*\\")"`
+      `"cdktf.Token.asString(cdktf.Fn.toset(\\"\${\\" + awsS3BucketExamplebucket.fqn + \\"}.*\\"))"`
```

Comment:
For this test case it doesn't really make sense to have a `getType` function that returns string 😄

Reply:
(this is also true for some other test cases)

Reply:
:) indeed. This is a bit noisy